scripts/xplatform-build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This script builds agent on different platforms
+
+set -ex
+
+CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -installsuffix cgo -a -ldflags "${LDFLAGS} -s" -o xplatform-ecs-agent ./agent/
```

Comment:
Can you invoke `./scripts/build` so you use the exact same flags and they stay in sync?